fuzz/fuzz_targets/fuzz_target_split_assemble.rs

```diff
@@ -0,0 +1,65 @@
+#![no_main]
+
+extern crate ctap2;
+extern crate libtock_drivers;
+#[macro_use]
+extern crate arrayref;
+
+use ctap2::ctap::hid::receive::MessageAssembler;
+use ctap2::ctap::hid::send::HidPacketIterator;
+use ctap2::ctap::hid::{HidPacket, Message};
+use libfuzzer_sys::fuzz_target;
+use libtock_drivers::timer::Timestamp;
+
+const DUMMY_TIMESTAMP: Timestamp<isize> = Timestamp::from_ms(0);
+const PACKET_TYPE_MASK: u8 = 0x80;
+
+// Converts a byte slice into Message
+fn raw_to_message(data: &[u8], len: usize) -> Message {
```

Comment:
The function seems to have redundant information. In Rust, a reference to a slice is a "fat pointer", i.e. it contains both a pointer to the slice and the length of the slice. So here, the `len` argument is redundant with `data`. You can replace all usages of `len` with `data.len()` in this function and remove the argument.